src/cdk/tree/control/base-tree-control.ts

```diff
@@ -9,7 +9,12 @@ import {SelectionModel} from '@angular/cdk/collections';
 import {Observable} from 'rxjs';
 import {TreeControl} from './tree-control';
 
-/** Base tree control. It has basic toggle/expand/collapse operations on a single data node. */
+/**
+ * Base tree control. It has basic toggle/expand/collapse operations on a single data node.
+ *
+ * @deprecated Use one of levelAccessor or childrenAccessor
+ * @breaking-change 14.0.0
```

Comment:
Might be good to leave a placeholder for these breaking changes version markers to make sure they are updated when merged to main

Reply:
done.